src/frontend/src/lib/components/core/Footer.svelte

```diff
@@ -13,31 +14,34 @@
 </script>
 
 <footer
-	class="z-1 mx-auto flex w-full max-w-screen-2.5xl flex-1 flex-col items-center justify-between px-4 pt-6 md:flex-1 md:flex-grow md:flex-row md:px-8"
+	class="z-1 mx-auto flex w-full max-w-screen-2.5xl flex-1 flex-col items-center justify-between px-4 pb-5 pt-6 md:flex-1 md:flex-grow md:flex-row md:px-8"
 	class:md:sticky={$authNotSignedIn}
 	class:md:h-md:mx-0={$authNotSignedIn}
 	class:md:h-md:w-[50%]={$authNotSignedIn}
-	class:md:h-md:pr-0={$authNotSignedIn}
 >
 	<div class="flex flex-row items-center gap-4">
 		<a
 			href={OISY_REPO_URL}
 			rel="external noopener noreferrer"
 			target="_blank"
-			class="rounded-full bg-white p-2"
 			aria-label={$i18n.navigation.text.source_code_on_github}
 		>
-			<IconGitHub />
+			<ButtonIcon ariaLabel={$i18n.navigation.text.source_code_on_github}>
```

Comment:
Mmmh a `button` within a `a`. Is that semantically speaking correct?

Reply:
I was in doubt a bit about this, but we will have some button styles that will be applied all over, even to link <a> elements.

So, I am not sure how to tackle this. The other solutions is to include such classes for both <button> and <a>, but I don't really like the idea somehow

Reply:
I should be a link. I think that from an accessibility and semantic point of view, using a button here, moreover stricly for styling purpose, is incorrect.

Reply:
Right, check last commit please: I created a new component for that, WDYT?